recipes/imfusion/0.3.2/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: 0.3.2
 
 build:
-  number: 0
+  number: 1
```

Comment:
`htslib` not pinned

Reply:
@mbargull not sure about these ones. I has probably a reason it is pinned this strict, we should not change it, should we?

Reply:
Yeah, like above, just don't change the build number as it's not needed here.